GameData/RealismOverhaul/RO_SAFConfig.cfg

```diff
@@ -0,0 +1,84 @@
+/// Example config - Replace 'MyPartWithSAFModule' with your part name
+/// @PART[MyPartWithSAFModule]:FOR[RealismOverhaul]
+/// {
+/// 	%ROSAFRescale = 1.81 //Make everything 1.81 times bigger
+/// }
+
+@PART[*]:HAS[@MODULE[ModuleSimpleAdjustableFairing]:HAS[#ROSAFRescale]]:AFTER[RealismOverhaul]
```

Comment:
This test does not match the comment.
Comment puts the tag at the PART level, but this patch looks for it inside the PartModule.
You want `@PART:HAS[@MODULE[ModuleSimpleAdjustableFairing],#ROSAFRescale];AFTER[RealismOverhaul] {...}`

Reply:
Fixed with latest commit, forgot to change that line when I moved the tag to the part level